src/org/opensolaris/opengrok/web/Util.java

```diff
@@ -590,7 +590,7 @@ public static void readableLine(int num, Writer out, Annotation annotation,
                 out.write("search\" href=\"" + env.getUrlPrefix());
                 out.write("defs=&amp;refs=&amp;path=");
                 out.write(project);
-                out.write("&amp;hist=" + URIEncode(r));
+                out.write("&amp;hist=&quot;" + URIEncode(r) + "&quot;");
```

Comment:
won't this break other SCMs ? can you test to index a sample git repo and try if history links work properly?

Reply:
This should not break other SCMs. All that is happening is that quotation marks are being placed around the revision number in the search form. In AccuRev the revision is of the form number/number and the '/' is a special character in the search system. Without the quotes, the user would get hollered at by OpenGrok. I think the other SCM's use just a simple number so normally this problem would not show up.  Here is a sample of the href line:

href="/source/s?defs=&refs=&path=&project=navigation&hist="1673%2F3"&type="
Here the %2F is the repesentation of '/'

How would I perform the test you are suggesting?  I am using netbeans 8.2 and all the documentation I find seems to be of older releases. The user interface has changed a lot and does not match the examples I am seeing.

Reply:
the test would be to index a git repo besides accurev and check the links in same places as accurev  (but for other history of course)

Reply:
OK, I ran this on a mecurial repository which gives a simple revision number with no ill effects.
As you can see by the enclosed picture.
![image](https://user-images.githubusercontent.com/13596544/30345978-36b5ccf0-97d5-11e7-8007-116c2061b729.png)